scrapy/utils/python.py

```diff
@@ -276,6 +280,7 @@ def equal_attributes(obj1, obj2, attributes):
 class WeakKeyCache:
 
     def __init__(self, default_factory):
+        warnings.warn("Call to deprecated Class WeakKeyCache", category=ScrapyDeprecationWarning, stacklevel=2)
```

Comment:
“Call” feels weird to me when talking about classes, and “class” should be lowercase.

:lipstick: What about…

```suggestion
        warnings.warn("The WeakKeyCache class is deprecated",
                      category=ScrapyDeprecationWarning, stacklevel=2)
```

Reply:
@Gallaecio resolved.